python/graphscope/framework/dag_utils.py

```diff
@@ -533,6 +533,32 @@ def to_undirected(graph):
     return op
 
 
+def get_graph_view(graph, view_type="reversed"):
```

Comment:
It is very wired to use "reversed" as a default view_type.  -- it leads to confusing result.

Reply:
fix with not use default value.